AGP-Kleros.md

```diff
@@ -0,0 +1,121 @@
+
+The only significant change would be to charge a subscription fee to organizations using the court even without disputes. The Kleros team believes that this change is likely to hinder adoption. But if governance were to decide so, it could be done without code modification simply by court policies asking jurors to vote “Refuse to arbitrate” on disputes of organizations who have not paid their subscription fees.
+
+#### Kleros community is open to change and experimentations
+
+Creating a new system when the community of a previous system is reluctant to change (as happened with Bitcoin and Ethereum) makes sense as it is the only way to innovate. But Kleros community is not reluctant to change. It is open to cryptoeconomic experiments which can be used to test proposed changes before implementing them at a larger scale. Note that most changes will be consensual and may not require extensive testing (most tweaks proposed by aragon members fit into this category).
+
+#### No benefits from competition to the ecosystem
+
+If the Aragon court were to use different mechanisms, it could benefit the ecosystem if those mechanisms were to perform better than Kleros ones and it would justify a new court creation. Competition is good when different products compete with each other as it gives people more choices and leads to better products more likely to “win” the market. But here, competition of similar products would just distract resources from both teams “fighting” each other for market share on grounds which are not product related.
+
+#### Creating competition with users is dangerous
+
+ Kleros uses Aragon for governance.
+
+Centralized platforms often use an attract-extract strategy.
+
+-   Attract: At the beginning platforms try to offer the best experience to their users, they are often subsidized with VC money. They welcome third parties (intermediate users) integrating and building on their platform.
+-   Extract: After reaching a critical mass of users. They now try to extract value from them. This includes competing with their users.
+
+<img width="396" alt="attract-extract" src="https://user-images.githubusercontent.com/8873352/63358602-3d5a0880-c363-11e9-8f13-fdf82840efd3.png">
+
+Decentralization is [*supposed to solve this problem*](https://medium.com/s/story/why-decentralization-matters-5e3f79f7638e). But it won’t if even decentralized platforms try to compete with their users.
+
+For Aragon, network effects among DAOs are not currently strong enough to control the ecosystem, so users are not bound to stay on Aragon. If Aragon were to compete with them, they could migrate their governance to other DAO platforms (DAOstack, MolochDAO forks or even develop their own) in order to avoid reinforcing a competitor. This would also scare would be users, as even if Aragon does not act as a competitor now, it could be in the future if it has a history of doing such things.
```

Comment:
```suggestion
For Aragon, network effects among DAOs are not currently strong enough to control the ecosystem, so users are not bound to stay on Aragon. If Aragon were to compete with them, they could migrate their governance to other DAO platforms (DAOstack, MolochDAO forks, or even develop their own) in order to avoid reinforcing a competitor. This would also scare would-be users, as even if Aragon does not act as a competitor now, it could be in the future if it has a history of doing such things.
```